srcpkgs/seatd/INSTALL.msg

```diff
@@ -0,0 +1,6 @@
+To use seatd you must ensure your user has permission to access
+seatd's socket.  The provided service runs seatd with the `-g seatd`
+option which sets the group owning the socket. Thus, you will most
+likely want to add your user to the seatd group:
+
+# usermod -aG seatd <username>
```

Comment:
Install messages are not documentation.

Reply:
The fact that the `seatd` group in particular is used is because of the `-g seatd` flag passed in the `run` file and is specific to void's package. Is that still not worth mentioning? I'd be happy to remove the install message as well.

Reply:
Install messages spam the output and users don't read them, adding more just makes users more unlikely to read them if there are actually important messages.

You could add a README.voidlinux file like the sqmail package does.

Reply:
Eh, I just removed the `INSTALL.msg`, I think this info is probably a better fit for the "Session and Seat Management" void docs page. The seatd man page and the 1 line `run` file should be plenty for most users to figure out how to use it.